### PR TITLE
Commands: create profile

### PR DESCRIPTION
**Pull request by amandate**
# Issue/Task:
https://fridgeventory.atlassian.net/browse/FRID-29

This is implementing the `create_profile()` method in commands. 

It should:
* prompt the user to name their profile
* create a new profile with the given name

This doesn't handle duplicate profile names yet.

# Checklist
- [x] Fixed Issue/Accomplished Task
- [x] Wrote tests
- [x] All test cases passing

### CODE REVIEW

File: tests/Session/test_commands.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+from src.Session.commands import Commands
+from unittest.mock import patch
```
Comment (leezahuor) [inline tests/Session/test_commands.py:2]: What does `patch` do and how is it being implemented in this test?

**Review by leezahuor — COMMENTED** [review]
Just one question, otherwise it looks good!

File: tests/Session/test_commands.py (hunk 1 above)
Comment (amandate) [reply tests/Session/test_commands.py:2]: It's used on line 13. I wanted a way to test when we request user input and found this way to do it. So I think how it works is it mocks the method you specify (in this case it's `input()` in `commands.py`) and then on line 16, we pass in what values we want as user input. 

So for example in `create_profile()`, we have this code:
```
# CREATE_PROFILE_NAME = "What would you like to name your profile?: "
profile_name = input(CREATE_PROFILE_NAME)
```

When the user is prompted with the question, our test passes in `"testProfile"` as a response, so `profile_name` will be set to `"testProfile"`

Comment (leezahuor) [reply tests/Session/test_commands.py:2]: I see, thank you very much for explaining!

**Review by leezahuor — APPROVED** [review]
Everything looks good!

**Pull request merged**